sdk/cosmosdb/cosmos/src/queryExecutionContext/EndpointComponent/GroupByEndpointComponent.ts

```diff
@@ -30,11 +31,19 @@ export class GroupByEndpointComponent implements ExecutionContext {
   public async nextItem(): Promise<Response<any>> {
     // If we have a full result set, begin returning results
     if (this.aggregateResultArray.length > 0) {
-      return { result: this.aggregateResultArray.pop(), headers: getInitialHeader() };
+      return {
```

Comment:
can we use type alias or interface to define the shape of the object that is being returned?

Reply:
The shape of the return object is defined by return type of method nextItem, i.e. Response